Corgibytes.Freshli.Lib/GitFileHistoryFinder.cs

```diff
@@ -117,10 +117,13 @@ string pattern
         )
         {
             string clonedProjectRoot = NormalizeLocation(projectRootPath);
-            return Directory.GetFiles(clonedProjectRoot,
-                                      pattern,
-                                      SearchOption.AllDirectories)
-                            .Select(f => Path.GetFileName(f)).ToArray();
+            var files = Directory.GetFiles(clonedProjectRoot,
+                                pattern,
+                                SearchOption.AllDirectories)
+                                .Select(f => f.Replace(clonedProjectRoot, ""))
+                                .Select(f => f.Substring(1))
```

Comment:
Why the `f.Substring(1)?` I'm sure there is a good reason but I'm just not smart enough to see it.

Reply:
I'll add a comment, but it clears the leading '/' in the filepath for the manifest file.

Reply:
Would [GetRelativePath](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.getrelativepath?view=net-5.0) work here?

Reply:
Changed to GetRelativePath.